examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p>Extremely unclear explanation!!!</p><p>Why anybody need to open the source of base TFModel or TorchModel?</p><p>You mention models, then TF and Torch, then again models and again Torch and TF, then models and once more TF and Torch. That's extremely confusing.</p><p>Write clearly, concisely and cohesively about models, TF / Torch difference and framework-agnosticism.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>